modules/shared/src/main/scala/org/tessellation/security/signature/SignedValidator.scala

```diff
@@ -105,6 +105,23 @@ object SignedValidator {
           .toNes
       }.map(SignersNotInSeedlist).toInvalidNec(signed)
 
+    def validateSignedByMajority[A](seedlist: Option[Set[PeerId]], signed: Signed[A]): SignedValidationErrorOr[Signed[A]] =
```

Comment:
`ByMajority` is misleading as if there is no seedlist, it's always valid. Either there should be a minimum no of signatures in case of no seedlist, or the fn name should be updated to sth like `validateSignedBySeedlistMajority`

Reply:
renamed